test/test_helper.rb

```diff
@@ -0,0 +1,7 @@
+require 'simplecov'
+
+SimpleCov.start
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'mocha/minitest'
```

Comment:
Added require mocha here instead of in the cipher_test because I believe there will be many more mocks and stubs used in other classes moving forward.